src/main/java/pb360/model/JsonDataModel.java

```diff
@@ -1,25 +1,27 @@
 package pb360.model;
-
 import java.util.List;
+
 import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonProperty;
+
 import pb360.model.embedded.JsonNode;
+
 import com.fasterxml.jackson.annotation.JsonInclude.Include;
 
 @JsonInclude(Include.NON_DEFAULT)
 public class JsonDataModel {
 	private String title;
-	private List<JsonNode> nodeList;
+	private List<JsonNode> Model;
```

Comment:
this is wrong, we have only a listNode, listNode will contain Model,Validator,Options...

Reply:
I will change it to listNode and put the other in them